src/expressjs/app.js

```diff
@@ -0,0 +1,21 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import { cookieParser } from './middlewares/cookieParser';
+import { queryParser } from './middlewares/queryParser';
+import userRouter from './routes/users/'
+import productRouter from './routes/products/'
+
+const app = express();
+
+app.use(cookieParser());
+app.use(queryParser());
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.use('/api/users', userRouter);
```

Comment:
Please, extract this to a separate root file like `router.js` and the export it and use as `app.use(router)`

Reply:
Thank, updated code.